test/run-php-linter.sh

```diff
@@ -17,5 +17,3 @@ vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php module
 
 # Run PHPCS on src/ directory using a different ruleset conforming to PSR2.
 vendor/bin/phpcs --standard=docs/SrcCS.xml --extensions=php/php src/ || exit $?;
-
-vendor/bin/phpmd php/libraries text docs/LorisPHPMD.xml || exit $?;
```

Comment:
This shouldn't be removed in this PR.

Reply:
Hey @johnsaigle, this is marked as Outdated and you made the comment 34 minutes ago after the commit that resolved this. Is there something still wrong with it? The file looks the same as major.

Reply:
Should be fine